cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/config/AbstractConfigTest.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+package com.linkedin.kafka.cruisecontrol.config;
+
+abstract class AbstractConfigTest {
```

Comment:
I am not sure if extending an abstract class like this by `SecurityAndSslConfigTest` and `WebserverUrlPrefixConfigTest` is the best approach. 
* `SecurityAndSslConfigTest` does not use `GET_CLASS_METHOD_NAME` anywhere, and 
* `WebserverUrlPrefixConfigTest` does not use `GET_BOOLEAN_METHOD_NAME` and `HASH_CODE_METHOD_NAME` anywhere.

Inheritance should imply that everything in the abstract class applies to the classes that extend from it. If the goal here is to share common constants across classes, can we move these constants to a utility class within `config` package?

Reply:
Sure